inference-engine/src/transformations/src/transformations/convert_previous_nms_to_nms_5.cpp

```diff
@@ -0,0 +1,254 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/convert_previous_nms_to_nms_5.hpp"
+
+#include <list>
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/opsets/opset3.hpp>
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/opsets/opset5.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::ConvertNMS4ToNMS5, "ConvertNMS4ToNMS5", 0);
+
+ngraph::pass::ConvertNMS4ToNMS5::ConvertNMS4ToNMS5() {
```

Comment:
Please, remove transformation inference-engine/src/transformations/src/transformations/convert_opset3_to_opset2/convert_nms3.cpp and inference-engine/src/transformations/src/transformations/convert_opset1_to_legacy/convert_nms_4_to_legacy.cpp since they are not needed any more.

Reply:
Deleted.